config/routes.rb

```diff
@@ -1,3 +1,6 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  resources :user, only: [:new, :create, :update, :edit]
+  get "users/new", to: "users#new"
+  post "users", to: 'users#create'
```

Comment:
If you replace `resources user` by **`resources users`**, you wont need this two routes anymore